src/components/Layout/Header/NotificationCenter/NotificationCenter.tsx

```diff
@@ -31,7 +34,36 @@ export const NotificationCenter = memo(() => {
   const translate = useTranslate()
   const handleToggleIsOpen = useCallback(() => setIsOpen(previousIsOpen => !previousIsOpen), [])
   const handleClose = useCallback(() => setIsOpen(false), [])
-  const hasPendingTxs = true
+
+  const actions = useAppSelector(selectInitializedActionsByUpdatedAtDescFilteredByWallet)
```

Comment:
preferably-blocking: at high-level, "actions" should be decoupled from wallet. Some (e.g swap things) may, but that's inferred, so we should probably ditch that from the name.

Reply:
That's a very nice catch, removed and used the sellAccountId from the swap by adding a selector that will do all the magic